docs/podman-run.1.md

```diff
@@ -653,6 +653,21 @@ Set the UTS mode for the container
 
 **NOTE**: the host mode gives the container access to changing the host's hostname and is therefore considered insecure.
 
+**−−mount**[=*type=<bind|tmpfs>,<src|source>=HOST−DIR,<target|dst,destination>=CONTAINER−DIR[:OPTIONS]*]
+The --mount flag allows you to mount host-directories and tmpfs mounts in a container.
+The --mount flag supports most options that are supported by the -v|--volume flag, but uses a different syntax.
+
+The type must be bind or tmpfs .tmpfs mounted folder data will be lost after container stops.
+
+The source can be specified as "source" or "src" & target can be specified as "target","dst" or "destination".
+The CONTAINER−DIR must be an absolute path such as /src/docs. The HOST−DIR must be an absolute path as well.
+podman bind −mounts the HOST−DIR to the path you specify.
+For example, if you supply the /foo value, podman creates a bind −mount.You can specify multiple −−mount options to mount
+one or more mounts to a container.
+
+You can add :ro or :rw suffix to a volume to mount it read −only or read −write mode, respectively. By default, the volumes are mounted read −write. See examples.
+
```

Comment:
I'm not a fan of "you" in man pages.  Can you please change that?  "For example if /foo is supplied as the value, Podman create a bind --mount.  Multiple --mount options can be specified to mount one or more mounts to a container.

The :ro or :rw suffix can be added to a volume ..." 

etc.